praw/models/reddit/subreddit.py

```diff
@@ -304,6 +420,29 @@ def unread(self, **generator_kwargs):
             self.subreddit._reddit, API_PATH['moderator_unread'].format(
                 subreddit=self.subreddit), **generator_kwargs)
 
+    def update_settings(self, **kwargs):
+        """Update the subreddit's settings.
+
+        See :meth:`Subreddit._create_or_update` for details on each parameter.
```

Comment:
As with `create`, can we explicitly list all the parameters in the docstring? They need not be explicitly listed in the function's signature, though.